bindings/matrix-sdk-crypto-js/tests/encryption.test.js

```diff
@@ -29,8 +29,11 @@ describe(EncryptionSettings.name, () => {
 
 describe('VerificationState', () => {
     test('has the correct variant values', () => {
-        expect(VerificationState.Trusted).toStrictEqual(0);
-        expect(VerificationState.Untrusted).toStrictEqual(1);
-        expect(VerificationState.UnknownDevice).toStrictEqual(2);
+        expect(VerificationState.SignedDeviceOfUnverifiedUser).toStrictEqual(1);
```

Comment:
`VerificationState.Verified` is missing here.

Reply:
thx updated